modules/installation-program-auto-create-subnets.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-aws-localzone.adoc
+
+:_content-type: PROCEDURE
+
+[id="installation-program-auto-create-subnets_{context}"]
+= Setting the installation program to automatically create subnets
+
+// This feature is related to the edge compute pool only?
```

Comment:
I feel that this module should be placed closer to the edge compute pool content? The automation feature extends to both remote zones and local zones?

https://docs.openshift.com/container-platform/4.13/installing/installing_aws/installing-aws-localzone.html#machines-edge-machine-pool_installing-aws-localzone

Reply:
Thanks. I'll move the content closer to this section tomorrow.

Reply:
Moved section close to existing edge compute pool content.